lib/utils.js

```diff
@@ -82,6 +82,9 @@ var retry = function retry(f, options) {
     return new Promise(function(accept) {
       accept(f(retry));
     }).catch(function(err) {
+      // Add 'fast-azure-storage' as module, so that errors can be traced
```

Comment:
You know what would be cool, is if Node had stack traces.. hahaha that'll never work! ;)